src/main/java/com/example/codeE/service/docker/DockerImpl.java

```diff
@@ -106,4 +110,17 @@ public Map<Integer, String> submitCode(String containerId, String fileName) {
         }
         return resultMap;
     }
+    @Override
+    public String RunCode(String containerId, String fileContent, String exerciseId) {
+        var language = codeExerciseService.getCodeExerciseById(exerciseId).getLanguage();
+        String output = "" ;
+        switch (language.toLowerCase()){
+            case "java":
+                output = docker.runCmd(containerId, "maven","run");
```

Comment:
I remember that the current template project is a normal java project that does not use maven. Are you sure this is correct?